packages/zoe/src/typeGuards.js

```diff
@@ -310,6 +312,7 @@ export const ZoeStorageManagerIKit = harden({
       IssuerPKeywordRecordShape,
       M.or(InstanceHandleShape, BundleShape),
```

Comment:
Why is the `instance` param `M.or(InstanceHandleShape, BundleShape)`?  That just seems wrong.

Reply:
Hi @Chris-Hibbert , in light of your comment, I just removed the automerge label and will look now. IIRC, I didn't change that one. What should it be?

Reply:
I'm guessing just `InstanceHandleShape` instead. I'll go ahead and make this change so we can see what CI thinks of it. But even if it is green, I'll wait for your reaction before proceeding.

Reply:
Just `InstanceHandleShape`.  I looked around, and the closest I could find was this line in `startInstance.js`
```
@param {Pick<ZoeStorageManager, 'makeZoeInstanceStorageManager' | 'unwrapInstallation'>} startInstanceAccess
```
I'll continue researching for a bit

Reply:
#4974 and #4975 make me doubt my instincts. I wouldn't change anything here now.

There seem to be places that insist that parameter must be an InstanceHandle, and others that insist that it must be a Bundle. The PRs I just referenced seem to expect the argument to `installBundle()` to be allowed to be any of multiple types, but I don't see how Zoe can be storing all of those under the name `instance` in the instanceRecord, which is how this parameter is being used.

Reply:
Ok. FWIW, at the present time, CI seems fine enough with this simplification

![image](https://github.com/Agoric/agoric-sdk/assets/273868/75f79cf4-bcf2-44d6-a6d5-27ef24f37a95)

But I'll back this out as you suggest, restart, and re-add the automerge label.

Please file an issue for this remaining question? Thanks.

Reply:
Backed out. Resolving.

Reply:
> Please file an issue for this remaining question? Thanks.

#8645